docroot/profiles/custom/cgov_site/modules/custom/cgov_migration/src/Plugin/migrate/process/RemoveRxWrapper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\cgov_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use PHPHtmlParser\Dom;
+
+/**
+ * Remove the rxbodyfield wrapper..
+ *
+ * @MigrateProcessPlugin(
+ *   id = "remove_rx_wrapper"
+ * )
+ */
+class RemoveRxWrapper extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+
+    // Process and return the value.
+    $dom = new Dom();
+    $dom->loadStr($value);
+
+    $contents = $dom->find('.rxbodyfield');
```

Comment:
Is there a way to:
1. Make sure that the rxbodyfield div was the first element.
2. Make sure there was only one.

Reply:
1. Partially resolved. There's an edgecase,  since the DomDocument generates an html/body wrapper if one doesn't exist in the incoming data, there's no straightforward way parse the raw value to  tell if something wonky is happening like adding the full page markup to the BODY field:

```
<html>
<body>
<div class="rxbodyfield"></div>
</body
</html>

```

2. Resolved.